EU4toV2/Source/V2World/Decisions/Decisions.cpp

```diff
@@ -0,0 +1,34 @@
+#include "Decisions.h"
+#include "ParserHelpers.h"
+#include "OSCompatibilityLayer.h"
+#include "Log.h"
+
+V2::Decisions::Decisions()
+{
+	const auto& decisionsFiles = commonItems::GetAllFilesInFolder("blankMod/output/decisions");
+	Log(LogLevel::Info) << "Loading decisions";
+	for (const auto& decisionsFile: decisionsFiles)
+	{
+		registerKeyword("political_decisions", [this](const std::string& unused, std::istream& theStream) {
+			Decisions newDecisions(theStream);
```

Comment:
you're instancing a new copy of this object for every file, but are not storing it anywhere. This can't work. Instead you need to either break this file apart into two objects - one global that goes through every file, and one that parses every file using Decision.cpp, or you need to store these individual newDecisions somewhere you can access them later.

Reply:
Ugh, yes, I had this in a later commit.